build.sh

```diff
@@ -2,6 +2,8 @@
 
 RELEASE=r6p0
 
+PARALLEL=
+
```

Comment:
And here initialising it to $(nproc) looks reasonable too.

We would need to change a bit the documentation, since our build would be parallel by default, but I don't really see any downside there.